samples/Builder.Filtering.Web/Startup.cs

```diff
@@ -47,8 +47,10 @@ public void Configure(IApplicationBuilder app)
         public static void Main(string[] args)
         {
             var host = new WebHostBuilder()
+                // We set the server by name before default args so that command line arguments can override it.
+                // This is used to allow deployers to choose the server for testing.
+                .UseServer("Microsoft.AspNetCore.Server.Kestrel")
```

Comment:
Should we use an expression like `typeof(KestrelServer).GetTypeInfo().Assembly.FullName` instead of hardcoding the string `"Microsoft.AspNetCore.Server.Kestrel"`?  The latter is more fragile if we ever rename the Kestrel assembly.